crypto/aes/asm/aes-s390x.pl

```diff
@@ -821,12 +821,9 @@ ()
 	ar	%r5,%r0
 
 	larl	%r1,OPENSSL_s390xcap_P
-	lg	%r0,0(%r1)
-	tmhl	%r0,0x4000	# check for message-security assist
-	jz	.Lekey_internal
-
 	llihh	%r0,0x8000
 	srlg	%r0,%r0,0(%r5)
+	ng	%r0,32(%r1)	# check km capability vector
 	ng	%r0,48(%r1)	# check kmc capability vector
```

Comment:
Could you modify the commentary clarifying that we check for both km and kmc for given key length, and rely on the fact that bit mask is same [for basic operations]. Trouble here is is that it's not self-obvious to fresh eye...

Reply:
I meant modify commentary specifically here, in above two lines. ng-ing two values leaves first-time reader wondering what is that about, suspecting it's a copy-n-paste error. But if commentary said "we check availability of both km and kmc support for given key length", i.e. one commentary describing both checks, then it wouldn't like like copy-n-paste error.

Reply:
okay, done.